hepcrawl/extractors/jats.py

```diff
@@ -83,7 +83,7 @@ def _clean_aff(self, aff):
         for el in root:
             if el.tag == 'label':
                 root.remove(el)
-        return ', '.join(root.itertext())
+        return ''.join(root.itertext())
```

Comment:
this was adding additional commas, after every parser part. Let's say our XML looks like this:
 ```<aff id="AFF1">Institute of Science and Engineering<institution>Shimane University</institution>,Matsue 690-8504, <country country="JP">Japan</country></aff>```

All the string values: `["Institute of Science and Engineering", "Shimane University", ",Matsue 690-8504, ", "Japan" ]`

join: `", ".join(["Institute of Science and Engineering", "Shimane University", ",Matsue 690-8504, ", "Japan" ])`

result: `'Institute of Science and Engineering, Shimane University, ,Matsue 690-8504, , Japan'`